models/index.js

```diff
@@ -1,6 +1,9 @@
 const Sequelize = require('sequelize')
 const dbPath = process.env.DATABASE_PATH || 'db/winder.db'
-const sequelize = new Sequelize(`sqlite://${dbPath}`, {
+const dbURL = process.env.DATABASE_URL || `sqlite://${dbPath}`;
```

Comment:
Maybe we can simply get rid of `dbPath`altogether and use the full descriptor (`sqllite://db/winder.db`) as a default value for `dbURL` ?